sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PodInfoBuilder.java

```diff
@@ -559,17 +548,18 @@ private static String getConfigTemplateDownloadPath(ConfigFileSpec config) {
                 && podSpec.getNetworks().isEmpty()
                 && podSpec.getRLimits().isEmpty()
                 && secretVolumes.isEmpty()) {
+            // Nothing left to do.
             return containerInfo.build();
         }
 
-        boolean shouldAddImage = podSpec.getImage().isPresent() && addExtraParameters &&
+        boolean shouldAddImage =
```

Comment:
I viscerally hate line breaks inside of if/loop condition blocks, so I elected to waste a byte of stack space to make it a little more readable.